### PR TITLE
Support Rails 6

### PR DESCRIPTION
**Pull request by shanecav84**
Wanted to test my own apps and needed to do this locally anyway. Might as well get this PR started.

### CODE REVIEW

**Comment by shanecav84** [comment]
Changes so far:

- Support Rails 6.0.0.beta
- Removed coveralls until [thor is updated](https://github.com/lemurheavy/coveralls-ruby/pull/132) because Rails 6 requires a higher version than coveralls allows
- Avoid the following deprecation warning for Rails version >= 6:
   ```
   DEPRECATION WARNING: `Module#parent_name` has been renamed to `module_parent_name`.
   `parent_name` is deprecated and will be removed in Rails 6.1.
```

File: docs/notifiers/google_chat.md [hunk 1]
```diff
@@ -27,4 +27,5 @@ The Incoming WebHook URL on Google Chats.
 
 *String, optional*
 
-Your application name, shown in the notification. Defaults to `Rails.application.class.parent_name.underscore`.
+Your application name, shown in the notification. Defaults to `Rails.application.class.parent_name.underscore` for Rails versions < 6;
+`Rails.application.class.parent_name.underscore` otherwise.
```
Comment (aried3r) [inline docs/notifiers/google_chat.md]: `Rails.application.class.module_parent_name.underscore`, right?

**Comment by ansonhoyt** [comment]
Are you good folks still working on this? Looks like you made good progress, and now that Rails 6.0.0.rc1 is out, you should have a solid target to land this against.

**Comment by shanecav84** [comment]
I've updated rails to 6.0.0.rc1 and rebased on master.

**Comment by EmilioCristalli** [comment]
thanks for the update @shanecav84 ! is this still a WIP or you think it's ready?

**Comment by shanecav84** [comment]
I don’t think anything will need to change between now and an official Rails 6 release other than rebasing.

**Comment by axlekb** [comment]
Ready for update! :-)

**Comment by shanecav84** [comment]
Finally 😄

**Comment by shanecav84** [comment]
@smartinez87 You might want to change the gemspec https://github.com/shanecav84/exception_notification/commit/201c8bec0f86e332fbd6cc251e3fbf4075b0e26f

**Pull request merged**